src/coin/Coin.ts

```diff
@@ -1,30 +1,34 @@
-import is from 'is_js';
+const is = require('is_js');
```

Comment:
After installing @types/is as devdependency
```sh
npm install --save-dev @types/is
```
```suggestion
import is from "is_js";
```